packages/map-template/src/components/Map/Map.jsx

```diff
@@ -117,6 +139,20 @@ function Map({ apiKey, gmApiKey, mapboxAccessToken, venues, venueName, onLocatio
         });
     }
 
+    /*
+     * React on changes in the tile style prop.
+     */
+    useEffect(() => {
+        if (tileStyle) {
+            _tileStyle = tileStyle;
+            onTileStyleChanged(mapsIndoorsInstance);
+        } else {
+            _tileStyle = 'default';
```

Comment:
Moved the `onTileStyleChanged` out of the if statement. However, the assignment of the `default` tile style is necessary, because i.e. we have the `tileStyle=3d_tiles`, but then if we comment/remove the tile style, we need to display the default tile style again.

Reply:
Got it.